src/events/eventToAction.js

```diff
@@ -88,11 +88,15 @@ export default (state: GlobalState, event: $FlowFixMe): EventAction => {
         ownEmail: getOwnEmail(state),
       };
 
-    case 'delete_message':
+    // Before server feature level 13, or if we don't specify the
+    // `bulk_message_deletion` client capability (which we do) this
+    // event has `message_id` instead of `message_ids`.
+    case 'delete_message': {
       return {
         type: EVENT_MESSAGE_DELETE,
-        messageId: event.message_id,
+        messageIds: event.message_ids ?? [event.message_id],
       };
+    }
```

Comment:
nit: extra braces not needed

(I'm guessing they came from a previous draft of this commit where you perhaps had a local variable.)

Reply:
Yep, that was it—thanks for catching this.